pom.xml

```diff
@@ -696,7 +696,6 @@
           </systemProperties>
           <reuseForks>true</reuseForks>
           <forkCount>${concurrency}</forkCount>
-          <argLine>-Djava.awt.headless=true ${argLine}</argLine>
```

Comment:
I think this is a regression, or at least a behavioral change. Previously, a custom `-DargLine=…` would have been _added_ on to `java.awt.headless`. Now it will quietly suppress `java.awt.headless`.

Not sure how this PR is supposed to work in detail, but it would seem safer to modify just the execution of the JaCoCo plugin.